src/pages/hegel/reference/becoming/development.mdx

```diff
@@ -0,0 +1,562 @@
+words, the thought demands one is immediate and the other is the reference.
+
+While this pattern of "passing into another" is valid for both, it plays out
+very differently whether it is `being` that passes into `nothing`, or `nothing`
+that passes into being. This is why Hegel states that the determinations in
+these unities are of unequal value.
+
+Once again, it is worth noting that Hegel has been using `determinate-` and
+`determination` in this section, and, like `negation`, these are terms not
+explicitly developed in the logic. Unlike `negation`, however, the terms at hand
+seem to do more work for the argument. On the one hand, determination here may
+not be used in a technical sense and merely makes the difference between `being`
+and `nothing` more poignant. On the other hand, however, the term stresses drive
+and resolution, and a sharp unity of fixed difference, which prefigures a later
+category. Its usage here is somewhat misleading, if not problematic, and Hegel
+could quite easily have replaced it with less technically charged words without
+affecting the argument.
+
+#### Emergence and Cessation
+
+> Becoming is in this way doubly determined. In one determination, nothing is
+> the immediate, that is, the determination begins with nothing and this refers
+> to being; that is to say, it passes over into it. In the other determination,
+> being is the immediate, that is, the determination begins with being and this
+> passes over into nothing &ndash; _coming-to-be_ and _ceasing-to-be_ (Hegel
+> 2010, 80/21.93).
+
+The two unities of `being` and `nothing` are recast into coming-to-be and
+ceasing-to-be based on which direction the passage occurs. The unequal value of
+these passages is perhaps more clear now in the rephrasing, since it spells out
+that it is being, in both accounts (coming-_to-be_ and ceasing-_to-be_), which
+either arrives or departs.
+
+What is key is that Hegel relates this back to `becoming`. It is in the context
+of movement that the inseparability of the two categories is made explicit, and
+any further details discovered will reflect back on this context.
+
+A possible complication to the logic is that the coming-to-be and
+ceasing-to-be&mdash; which currently is defined with the vanishing of `being`
+into `nothing` and `nothing` into `being` respectively&mdash;could be equally
+applied over the category `nothing` itself. That is to say, `nothing` could be
+coming-to-be where _its_ self-subsistence _as nothing_ concerned and
+creasing-to-be when _it_ vanishes into `being`. This idea, while tempting at
+first, is not really coherent. Firstly, it presupposes the original coming-to-be
+and ceasing-to-be vis-à-vis `being` and `nothing`; secondly, it breaks the
+definition of `nothing` as the absolute absence of `being`, which is required
+for the original movement. Therefore, the coming-to-be and ceasing-to-be cannot
+apply to `nothing` except in a very abstract or non-substantive sense.
```

Comment:
I'm not sure I agree with this analysis. I think the section you quote shows that it is interchangeable, that nothing can be said to be coming-to-be and being can be said to be ceasing-to-be. Hegel calls it a double determination and outlines how nothing can be thought of as the immediate and being the thing to which it refers, _i.e._ it can be thought of as coming-to-be.

Reply:
Well, it appears we disagree on this. I first was thinking along the lines you are now, but it didn't make sense to me. What do you think about the points I make about it at the end of the paragraph?

Reply:
I have read the text carefully again, and you're right. Hegel clearly identifies coming-to-be with the transition from nothing to being, and ceasing-to-be with the transition from being to nothing. What is interchangeable is where we begin, either from coming-to-be or ceasing-to-be. Which, interestingly, is also what Hegel says about the beginning of the SL that it could have begun with nothing (Miller, 100), which echoes this idea.